src/components/add-user/index.tsx

```diff
@@ -24,6 +26,17 @@ export const AddCreateUser = ({ onSuccess }: AddUserProps) => {
     invalidBirthDate: '',
   });
 
+  const token = localStorage.getItem('token');
```

Comment:
essa variável é um estado? se não for o useEffect nunca vai ser ativado.

Reply:
obrigado pela observação! não é um estado.